packages/ts/src/components/treemap/config.ts

```diff
@@ -0,0 +1,34 @@
+import { ComponentConfigInterface, ComponentDefaultConfig } from 'core/component/config'
+import { TreemapNode } from 'types'
+import { ColorAccessor, NumericAccessor, StringAccessor } from 'types/accessor'
+
+export interface TreemapConfigInterface<Datum> extends ComponentConfigInterface {
+  id?: ((d: Datum, i: number) => string | number);
+  /* Numeric accessor for segment size value. Default: `undefined`. */
+  value?: NumericAccessor<Datum>;
+
+  /** Array of accessor functions to defined the nested groups  */
```

Comment:
Needs a default to be specified

Reply:
Also applies to nested donut implementation.

Will add defaults to the docs in both places.